hack/jenkins/minikube_cross_build_and_upload.sh

```diff
@@ -41,6 +41,7 @@ declare -rx DEB_VER="$(make deb_version)"
 docker kill $(docker ps -q) || true
 docker rm $(docker ps -aq) || true
 docker system prune -a --volumes -f
+docker login -u ${DOCKERHUB_USER} -p ${DOCKERHUB_PASS}
```

Comment:
This exposes the password in the environment, and in ps output. Currently the environment is logged in the build log, accessible to the world.

Maybe store it in a user readable file?

Reply:
I changed the password to Readonly Accesstoken